packages/backend/src/config/app.ts

```diff
@@ -88,8 +88,6 @@ export const Config = {
   incomingPaymentWorkers: envInt('INCOMING_PAYMENT_WORKERS', 1),
   incomingPaymentWorkerIdle: envInt('INCOMING_PAYMENT_WORKER_IDLE', 200), // milliseconds
 
-  quoteUrl: envString('QUOTE_URL', 'http://127.0.0.1:4001/quote'),
```

Comment:
Can you remove that from the integration docs, please?

Reply:
Looks like there were a few places to remove that. I also updated the explanation for setting fees in the integration doc https://github.com/interledger/rafiki/pull/1755/commits/e6f6f340651a97455094a1efd3316487481583c4